model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1093,6 +1093,72 @@ message SideInput {
   FunctionSpec window_mapping_fn = 3;
 }
 
+message StandardArtifacts {
+  enum Types {
+    // A URN for artifacts stored in a local directory.
+    // payload: ArtifactFilePayload.
+    FILE     = 0 [(beam_urn) = "beam:artifact:file:v1"];
+
+    // A URN for artifacts embedded in ArtifactInformation proto.
+    // payload: EmbeddedFilePayload.
+    EMBEDDED = 1 [(beam_urn) = "beam:artifact:embedded:v1"];
+
+    // A URN for artifacts described by remote URLs.
+    // payload: a string for an artifact URL e.g. "https://.../foo.jar"
+    REMOTE     = 2 [(beam_urn) = "beam:artifact:remote:v1"];
```

Comment:
Not needed it we adopt ArtifactUrlPayload

Reply:
Regarding the file payload, I think we may want to stage this under a different name than its "real" name. However, I think the stage name needs to be lifted to a higher level and apply to all artifact types. 

One reason to keep file separate than URL is that we may want to set the space of files be those understood by (and fetched via) the beam filesystems packages (including remote, non-(public?)-url ones like gcs or aws).

Reply:
I would call this one URL rather than REMOTE as most of these are remote.

Reply:
`FILE` and `REMOTE` are merged and renamed to `URL`.